style.css

```diff
@@ -313,3 +318,214 @@ textarea {
   color: #42526e;
   margin-bottom: 147px;
 }
+
+/* -----------------------------------media */
+
+@media screen and (min-width: 1080px) {
+  body {
+    background-image: url(imege/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 100%;
+  }
+
+  nav {
+    height: 32px;
+  }
+
+  .nav-list {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem;
+    gap: 40px;
+    list-style: none;
+    font-weight: 500;
+    height: 100px;
+    padding-bottom: 6rem;
+  }
+
+  nav ul li a {
+    color: #42526e;
+  }
+
+  .nav-list :hover {
+    color: #36b37e;
+    transition: 0.3s;
+  }
+
+  .nav-list :active {
+    color: #42526e;
+  }
+
+  .logo,
+  .menu {
+    display: none;
+  }
+
+  .first-page {
+    background-image: none;
+    padding: 160px 0 0 0;
```

Comment:
- [ ] You did an amazing job adding the desktop version for the first section but please add the arrow like in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) because it is hiding. Keep rocking 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/160027342-befd62e4-13d1-4819-95d8-7432e87d5ab2.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/160027305-3ff5d204-6ee9-45f8-926b-732b279a9692.png)

Reply:
I did it like the Figma desktop version, the arrow in Figma design is just in the mobile version not on desktop